wire-ios/Wire-iOS/Sources/UserInterface/Settings/CellDescriptors/SettingsSignOutCellDescriptor.swift

```diff
@@ -54,7 +56,7 @@ final class SettingsSignOutCellDescriptor: SettingsExternalScreenCellDescriptor
             AVSMediaManager.sharedInstance()?.stop(sound: .ringingFromThemSound)
             ZMUserSession.shared()?.logout(credentials: UserEmailCredentials(email: "", password: password ?? "")) { [weak topMostViewController] result in
                 Task { @MainActor in self.activityIndicator.stop() }
-                TrackingManager.shared.disableAnalyticsSharing = false
+                self.trackingManager.disableAnalyticsSharing = false
```

Comment:
Why does logging out change if analytics is enabled?

Reply:
It was there in the previous implementation but for the new one we don't have anything that explicitly says we need to do anything regarding analytics during log out. And because of that I removed the tracking manager entirely from the descriptor: https://github.com/wireapp/wire-ios/commit/a8288ec6ca5c59eacd5afbfa2b32558f2ac9deac